gramex/apps/capture/chromecapture.js

```diff
@@ -11,6 +11,8 @@ const path = require('path')
 const tmp = require('tmp')
 const fs = require('fs')
 const _ = require('lodash')
+const url = require('url')
```

Comment:
I believe this won't be required. In a comment below, I suggest that we just use `q._test_disconnect` instead of `url.parse`